metricbeat/module/apache/status/status.go

```diff
@@ -87,10 +92,36 @@ func (m *MetricSet) Fetch(reporter mb.ReporterV2) error {
 	}
 
 	data, _ := eventMapping(scanner, m.Host())
+	event := mb.Event{
+		MetricSetFields: data,
+	}
 
-	if reported := reporter.Event(mb.Event{MetricSetFields: data}); !reported {
-		m.Logger().Error("error reporting event")
+	if m.isFleetMode {
+		event = adjustFleetEvent(event)
 	}
 
+	if reported := reporter.Event(event); !reported {
+		m.Logger().Error("error reporting event")
+	}
 	return nil
 }
+
+func adjustFleetEvent(event mb.Event) mb.Event {
+	var adjusted mb.Event
+	adjusted.MetricSetFields = event.MetricSetFields.Clone()
+
+	// Remove apache.hostname
+	adjusted.MetricSetFields.Delete("hostname")
+
+	// Replace "apache.uptime" with "uptime"
+	uptime, err := adjusted.MetricSetFields.GetValue("uptime.uptime")
+	if err == nil {
+		if adjusted.RootFields == nil {
+			adjusted.RootFields = common.MapStr{}
+		}
+
+		adjusted.RootFields.Put("host.uptime", uptime)
+		adjusted.MetricSetFields.Delete("uptime")
```

Comment:
Take into account that `uptime` fields are also used in dashboards.
https://github.com/elastic/beats/blob/af9788859a7ddd7df4b1677fc43b7533d5245f3b/metricbeat/module/apache/_meta/kibana/7/dashboard/Metricbeat-apache-overview.json#L503
https://github.com/elastic/beats/blob/af9788859a7ddd7df4b1677fc43b7533d5245f3b/metricbeat/module/apache/_meta/kibana/7/dashboard/Metricbeat-apache-overview.json#L512

Reply:
That's why the change is introduced only in the fleet mode. I will adjust dashboards dedicated to Integrations in https://github.com/elastic/integrations/issues/370 .